src/lang/en.json

```diff
@@ -49,6 +49,9 @@
   "gold_reasons_14": "Roshan",
   "gold_reasons_15": "Courier",
 
+  "radiant_ability_builds": "Radiant Ability Builds",
```

Comment:
you should probably just build the string with (Radiant|Dire) + existing tooltip.